lib/tasks/show-latest-id.js

```diff
@@ -0,0 +1,23 @@
+/* jshint globalstrict:false, strict:false, sub: true */
+/* global print */
+exports.name = "show-latest-id";
+exports.group = "latest id";
+exports.args = [
+];
+
+exports.args_arangosh = " --server.endpoint AGENT-OR-COORDINATOR";
+exports.description = "Show the /arango/Sync/LatestID.";
+exports.selfTests = ["arango", "db", "leaderAgencyConnection"];
+exports.requires = "3.3.23 - 3.10.99";
+exports.info = `
+Read the entry /arango/Sync/LatestID.
+`;
+
+exports.run = function (extra, args) {
+  const helper = require('../helper.js');
+  const data = [['/arango/Sync/LatestID']];
+
+  const res = helper.httpWrapper('POST', '/_api/agency/read', data);
+
+  print("INFO: " + JSON.stringify(res));
```

Comment:
Intentional to print this whole object? Why not just print the number?

Reply:
Adjusted so that it now only shows the latest id value, not the entire JSON.